tests/suites/test_suite_ctr_drbg.function

```diff
@@ -137,10 +137,12 @@ void ctr_drbg_validate_no_reseed(data_t *add_init, data_t *entropy,
                                  data_t *result_string)
 {
     data_t empty = { 0, 0 };
+    USE_PSA_INIT();
```

Comment:
I don't think that's the right pair of macros. Use of PSA by CTR-DRBG should not depend on whether `USE_PSA` is on, but on whether `AES_C` is. So, we probably need yet another pair of macros, in addition to the existing `USE_PSA_INIT/DONE`, `MD_PSA_INIT/DONE`, `MD_OR_USE_PSA_INIT/DONE`. I'd say `AES_PSA_INIT/DONE` would be an OK name (similar to `MD_PSA_INIT/DONE`).